docs/team_strategy/facilitator_notes.md

```diff
@@ -0,0 +1,69 @@
+* **Stakeholder engagement and review**: The most important part of pre-session work. Ensure you understand the needs, wants and aims of your stakeholders, as well as their approach to this work. If they are fully bought in and have time, you can likely run more, in depth sessions. If they are busy/not fully bought in, you might need to pick on or two key sessions to run. You should also get feedback on your proposed strategy plans ahead of any session, to ensure co-creation of team strategy from the very beginning.
+* **Number of facilitators**: We recommend ensuring there are at least two facilitators for any session - one to chair the session, and one to help with note-taking, assigning breakout rooms, identifying people who haven't shared much etc.
+
+### :bulb: Insights
+* **Empowering/equal voices**: Set up the sessions from the very beginning to focus on empowerment of team members. This can include only speaking with a raised hand, setting a standard of maximum time to speak at one time (e.g. 2 minutes), prioritising those who have not shared much, having multiple ways to share thoughts (speaking, in a chat, on a document or note). Getting this wrong can mean people not buying into the strategy or feeling they are part of it, which can be disastrous for building a cohesive strategy.
+* **Focus on live facilitation**: This guide is relatively open and unstructured, with a strong focus on effective in-person facilitation. We find this to be a good approach, as it means momentum can be built live within the workshop, rather than asynchronously afterwards. However we understand this is more resource intensive, so please adapt the approach as best fits your need!
+* **Review only what was covered in sessions**: When synthesizing/clarifying discussions between sessions (see below), it is crucial to ensure no new context/information is created or included. This ensures that the team is fully involved with all aspects of how the strategy is created, and the process of creation is truly co-creative and transparent.
+* **Give conversations the time they need**: Linked to the point below, but we find it most effective to allow conversations the time they need - if there is strong disagreement, prioritise moving towards a place of alignment over getting all parts of the session completed.
+* **Checkpoints/don't revisit approach**: If alignment is reached, you should have natural checkpoints at the end of each session. If done effectively, it means you should be able to reach a new foundation after each session, which doesn't need to be revisited. Being firm on not revisiting old points will help ensure continued progress in each session.
```

Comment:
I think I would benefit from understanding further the value of "don't revisit" and what problem that addresses

Reply:
Made some edits to try and clarify